zebra/zebra_srv6_vty.c

```diff
@@ -371,6 +374,38 @@ DEFPY (locator_prefix,
 	return CMD_SUCCESS;
 }
 
+DEFPY (locator_behavior,
+       locator_behavior_cmd,
+       "[no] behavior usid",
+	   NO_STR
```

Comment:
Seems bad indentation.

Reply:
Something went wrong with tabs and spaces :)
Fixed, thanks.